bin/magento

```diff
@@ -19,7 +19,7 @@ try {
 try {
     $handler = new \Magento\Framework\App\ErrorHandler();
     set_error_handler([$handler, 'handler']);
-    $application = new Magento\Framework\Console\Cli('Magento CLI');
+    $application = new \Magento\Framework\Console\Cli('Magento CLI');
```

Comment:
Try `Alt+Enter` + `Import Class` in PhpStorm ;)

Hope to see `T_NAMESPACE_SEPARATOR` banned anywhere besides `namespace` and `use` statements someday...